src/libcip54.ts

```diff
@@ -766,6 +1284,13 @@ export const getFiles = async (
   }
   return files;
 };
+/**
```

Comment:
After the getFiles function above we seem to have got confused and stopped fully processing results.